cr-02-08-2022

Ami777 opened the pull request.

agencik007 committed:
Init branch commit

agencik007 committed:
Added search input and filter modal

agencik007 committed:
Added action on search form, renamed styles

agencik007 committed:
Fixed margins and paddings

agencik007 committed:
Small styling fixes

agencik007 committed:
Styling fixes

agencik007 committed:
chnaged width for search input

MS151994 committed:
Feature/refactor (#23)

* Move style to component folder.
Edit: box shadow
Add font size

* Move style to component folder.
Change property props width OR default variable

* Remove tests

* Add font size property to button style

* Remove unused filter modal

* Change folder name and edit some style

* Move Logo component to commons folder
refactor some style

* Move Logo component to commons folder
refactor some style

* Remove logo element - old component

* Add value from const description and refactor style

* Remove unused component

* Add 404 page component

* Refactor form style. adjust to display on mobile and tablet

* Add test page

* Add react-json-view

agencik007 committed:
Merge remote-tracking branch 'origin/develop' into develop

# Conflicts:
#	src/components/recruiters/Navigation/Navigation.tsx
#	src/pages/Home.page.tsx
#	yarn.lock

RAlbert5477 committed:
Added Admin component

Fremen1990 committed:
Shared types config BE vs FE added

mateuszslowinski committed:
Merge pull request #24 from Fremen1990/feature/BackendApiCalls-Admin-tests

Feature/backend api calls admin tests

Fremen1990 committed:
Merge pull request #25 from Fremen1990/feature/15-BE/FE-Types-sharing

Shared types config BE vs FE added

mateuszslowinski committed:
Feature/11.student portfolio (#26)

* Feature/11.StudentPortfolio

* Created and styled aside section.

* Created and styled main section.

* Created LinksBox and styled.

* Created AsideUserBox and styled it.

* Created SubtitlesSection component and styled it.

* Created DescriptionBox and styled it.

* Install react-star-ratings

* Created RatingVewBox component.

* Created RatingBox component.

* Created ExpectationBox component.

* Add props to LinksBox to fix shows SubtitlesSection to many times.

* Updated structure StudentPortfolio and styles.

* Updated LinksBox css.

* Updated DescriptionBox css.

* Add new constants colors.

* Updated RatingViewBox css.

* Added new constants description.

* Updated AsideBox css and added new constant.

* Updated ExpectationBo for new constant.

* Updated RatingBox for new constant.

* Updated StudentPortfolio css and added new constant.

* Created ExpectationBox styles file.

* Adjusted mobile view.

* Added Link to back to main page.

* Added new routes and used to show student cv page.

Co-authored-by: Mateusz <nowe7246@gmail.com>

Commit:
Improved display of the first letter to uppercase.

Commit:
Changes from Link component to Back button

Commit:
Added student portfolio modal in hr page.

Commit:
Remove student portfolio page.

agencik007 committed:
Merge remote-tracking branch 'origin/develop' into develop

Commit:
Repair showing student portfolio modal to show all necessary components.

Commit:
Init edit student portfolio modal.

Fremen1990 committed:
Merge pull request #28 from Fremen1990/feature/11.student-portfolio-updated

Feature/11.student portfolio updated

Commit:
Add useState to navigation to change tabs between students.

Commit:
Add students interface and get students from api.

Commit:
Use props to show student data on hr page and students modal.

Commit:
Add students data to ExpectationBox.

Commit:
Add show data to DescriptionBox.

Commit:
Add show data to LinksBox.

Commit:
Create form structure with styles.

Commit:
Added EditExpectationBox component.

Fremen1990 committed:
Merge branch 'develop' into feature/44.admin-page

Fremen1990 committed:
Merge pull request #27 from Fremen1990/feature/44.admin-page

Added Admin component

Fremen1990 committed:
Merge pull request #31 from Fremen1990/feature/basic-logic-for-show-to-customer

Feature/basic logic for show to customer

Fremen1990 committed:
Redux Toolkit initialized and login with global state implemented.

Fremen1990 committed:
Redux Toolkit login, logout, getProfile

agencik007 committed:
Merge branch 'develop' of https://github.com/Fremen1990/head-hunter-frontend into develop

Commit:
Created InputTextBox.

Commit:
Created NumberInputBox.

Commit:
Created SelectBox.

Commit:
Created TextAreaBox.

Commit:
Refactor structure of form.

Commit:
Options moved to separate file selectOptions.

Commit:
Moved EditStudentPortfolio to modals file and add BackNavLink to user page.

Commit:
Adjust the EditStudentPortfolio modal to the view and changed back navLink to button.

mateuszslowinski committed:
Merge branch 'develop' into feature/57.edit-student-portfolio

Commit:
Created UrlBox and refactored code.

Fremen1990 committed:
Redux Toolkit login, logout, getProfile - credentials

Commit:
Add correct types to selectOptions.

Commit:
Merge remote-tracking branch 'origin/feature/57.edit-student-portfolio' into feature/57.edit-student-portfolio

Commit:
Created types file and moved Cv interface there.

Commit:
Created EditStudentPortfolioModal and refactor code.

raczko committed:
Added AddHumanResources component
Feature/54.admin-page-form

raczko committed:
Create the form input
Feature/54.admin-page-form

RAlbert5477 committed:
Change component file name AddHumanResources.tsx on AddHrForm.tsx
Feature/54.admin-page-form

raczko committed:
Set limits and input types for the component Form
Feature/54.admin-page-form

raczko committed:
Added Styles to component
Feature/54.admin-page-form

Fremen1990 committed:
Redux Toolkit login, logout, getProfile - credentials, home page and other :)

Fremen1990 committed:
Merge pull request #33 from Fremen1990/feature/17.FE-LOGIN-Redux-toolkit-protected-routes

Feature/17.fe login redux toolkit protected routes

mateuszslowinski committed:
Merge pull request #32 from Fremen1990/feature/57.edit-student-portfolio

Feature/57.edit student portfolio

Fremen1990 committed:
Login bugfix

agencik007 committed:
Merge remote-tracking branch 'origin/develop' into develop

Fremen1990 committed:
Header redux data test

RAlbert5477 committed:
Fixed component, refactored styled
Feature/54.admin-page-form

Fremen1990 committed:
HOTFIX - bug with protected routes. Addidional refactor from ground of redux, one more hrSlice, some additional pages and refactoring

Fremen1990 committed:
HOTFIX - bug with protected routes. Addidional refactor from ground of redux, one more hrSlice, some additional pages and refactoring

Fremen1990 committed:
Logo assets

Fremen1990 committed:
Logo assets

Fremen1990 committed:
HotFix - not working login protected routes between Admin and HR, incorrect destructurization

Ami777 commented on .eslintrc.json:
```diff
@@ -21,5 +21,7 @@
         "@typescript-eslint"
     ],
     "rules": {
+//        "no-unused-vars": "off"
```
Dlaczego? To dobra zasada

Fremen1990 replied:
dodałem to sam do swojego lokalnego repo i komentuje jak robie push na remote, nie wiedziałem czy to legalna zasada :)

Ami777 commented on src/App.tsx:
```diff
@@ -4,20 +4,37 @@ import * as ROUTES from './constants/routes'
+import { RegisterPage } from './pages/Register.page'
+import { HomePage } from './pages/HomePage'
+
+export interface IAppProps {}
```
Raczej w TS staramy się nie zaczynać interfejsów od `I`

Ami777 commented on src/App.tsx:
```diff
@@ -4,20 +4,37 @@ import * as ROUTES from './constants/routes'
-            <Route path={ROUTES.HR_PAGE} element={<HrPage />} />
-            <Route path={ROUTES.USER_PAGE} element={<UserPage />} />
-            <Route path={ROUTES.ADMIN_PAGE} element={<AdminPage />} />
+            <Route path={ROUTES.HOME} element={<HomePage />} />
```
Czy jest jakiś powód, dla którego jest to w osobnym miejscu?

Ami777 commented on src/components/EditStudentPortfolio/UrlBox/UrlBox.tsx:
```diff
@@ -0,0 +1,34 @@
+   btnMethod,
+   btnText,
+}: Props) => {
+   return (
```
Jak tylko tyle, to można skrócić. Dać u góry `(`, na dole `)` i bez `return`.

Ami777 commented on src/components/EditStudentPortfolio/UrlBox/UrlBox.tsx:
```diff
@@ -0,0 +1,34 @@
+      <label>
+         <SubtitlesSection text="Projekt na zaliczenie" />
+         {urlBoxArray.map((project: string, index: number) => (
+            <p key={index}>{project}</p>
```
Mam wrażenie, że tu lepszym kluczem bedzie `project`, bo powinien być raczej unikalny.

Ami777 commented on src/components/UserForm/Login.form.tsx:
```diff
@@ -28,15 +33,22 @@ export const LoginForm = () => {
+   const onSubmit = async (data: Login) => {
+      const { email, password } = data
+      if (email && password) {
+         // @ts-ignore
```
Dlaczego?

Fremen1990 replied:
dispatch i Redux toolkit, nie ogarnąłem jeszcze jak otypować async thunki

Ami777 commented on src/pages/Hr.page.tsx:
```diff
@@ -1,16 +1,120 @@
+   workExperience: string
+   courses: string
+   created_at?: Date
+   updated_at?: Date
```
Znowu brak konsekwencji w nazwach

Ami777 reviewed (commented):
Bardzo ładny i nowoczesny kod. W komentarzach kilka podpowiedzi

Fremen1990 committed:
HotFix - not working login protected routes between Admin and HR, incorrect destructurization

RAlbert5477 committed:
Refactored component
Feature/54.admin-page-form

agencik007 committed:
Merge remote-tracking branch 'origin/develop' into develop

agencik007 committed:
Added navigation bar

agencik007 committed:
Updated description file

agencik007 committed:
Hr page fixed imports

agencik007 committed:
Fixed filter padding

agencik007 committed:
Merged views in students page with mock data - getting candidates from db is not supported yet

RAlbert5477 committed:
Merge branch 'develop' into feature/54.admin-page-form

mateuszslowinski committed:
57.edit student portfolio updated (#37)

* Add validation to edit modal student cv.

* A new constant description has been added and used in modal cv edition.

* Moved types file to src folder.

* Refactor code of UrlBox.

Co-authored-by: Mateusz <nowe7246@gmail.com>

Fremen1990 committed:
Merge pull request #36 from Fremen1990/feature/54.admin-page-form

Feature/54.admin-page-form

MS151994 committed:
Feature/10.filtering modal (#29)

* Created filtering views

* Separate filter form

* Init edit student portfolio modal.

* Create form structure with styles.

* Added EditExpectationBox component.

* Separate one element to component folder

* Add types to event variable

* Add FilterReducer

* Add test page and add filtering modal

* Refactor name components

* Add button Input

* Update description text

* Move tab of filter element to separate folder

* Move reducer to Utils folder

* Add salary to reducer

* Refactor style

* Refactor style add outline color

* Refactor style

* Refactor code, add salary input, internship,experience input.

* Add internship

* Redux Toolkit initialized and login with global state implemented.

* Redux Toolkit login, logout, getProfile

* Created InputTextBox.

* Created NumberInputBox.

* Created SelectBox.

* Created TextAreaBox.

* Refactor structure of form.

* Options moved to separate file selectOptions.

* Moved EditStudentPortfolio to modals file and add BackNavLink to user page.

* Adjust the EditStudentPortfolio modal to the view and changed back navLink to button.

* Created UrlBox and refactored code.

* Redux Toolkit login, logout, getProfile - credentials

* Add correct types to selectOptions.

* Created types file and moved Cv interface there.

* Created EditStudentPortfolioModal and refactor code.

* Added AddHumanResources component
Feature/54.admin-page-form

* Create the form input
Feature/54.admin-page-form

* Change component file name AddHumanResources.tsx on AddHrForm.tsx
Feature/54.admin-page-form

* Set limits and input types for the component Form
Feature/54.admin-page-form

* Added Styles to component
Feature/54.admin-page-form

* Redux Toolkit login, logout, getProfile - credentials, home page and other :)

* Update description

* Add export initial state for Reducer

* Add Clear button

* Refactor component

* Refactor types

* Add mobile views

* Login bugfix

* Header redux data test

* Fixed component, refactored styled
Feature/54.admin-page-form

* HOTFIX - bug with protected routes. Addidional refactor from ground of redux, one more hrSlice, some additional pages and refactoring

* HOTFIX - bug with protected routes. Addidional refactor from ground of redux, one more hrSlice, some additional pages and refactoring

* Logo assets

* Logo assets

* HotFix - not working login protected routes between Admin and HR, incorrect destructurization

* HotFix - not working login protected routes between Admin and HR, incorrect destructurization

* Refactored component
Feature/54.admin-page-form

* 57.edit student portfolio updated (#37)

* Add validation to edit modal student cv.

* A new constant description has been added and used in modal cv edition.

* Moved types file to src folder.

* Refactor code of UrlBox.

Co-authored-by: Mateusz <nowe7246@gmail.com>

* Created filtering views

* Separate filter form

* Separate one element to component folder

* Add types to event variable

* Add FilterReducer

* add filtering modal

* Refactor name components

* Add button Input

* Update description text

* Move tab of filter element to separate folder

* Move reducer to Utils folder

* Add salary to reducer

* Refactor style

* Refactor style add outline color

* Refactor style

* Refactor code, add salary input, internship,experience input.

* Add internship

* Update description

* Add export initial state for Reducer

* Add Clear button

* Refactor component

* Refactor types

* Add mobile views

* Created filtering views

* Separate filter form

* Separate one element to component folder

* Add types to event variable

* Add FilterReducer

* add filtering modal

* Refactor name components

* Add button Input

* Update description text

* Move tab of filter element to separate folder

* Move reducer to Utils folder

* Add salary to reducer

* Refactor style

* Refactor style add outline color

* Refactor style

* Refactor code, add salary input, internship,experience input.

* Add internship

* Add export initial state for Reducer

* Add Clear button

* Refactor component

* Refactor types

* Add mobile views

* Separate one element to component folder

* Add types to event variable

* Add FilterReducer

* Add test page and add filtering modal

* Refactor name components

* Update description text

* Add salary to reducer

* Add export initial state for Reducer

* Refactor types

* Fix filter

* Refactor types

Co-authored-by: Mateusz <nowe7246@gmail.com>
Co-authored-by: DevThomas <fremen1990@gmail.com>
Co-authored-by: mateuszslowinski <105711183+mateuszslowinski@users.noreply.github.com>
Co-authored-by: raczko <raczekw11@wp.pl>
Co-authored-by: RAlbert5477 <rogoz.albert2003@gmail.com>
Co-authored-by: Tomasz (DevThomas) <56457689+Fremen1990@users.noreply.github.com>
Co-authored-by: albert_rogoz <105876587+RAlbert5477@users.noreply.github.com>

MS151994 committed:
Revert "Feature/10.filtering modal (#29)" (#38)

This reverts commit a63ef9b0d6576fc3009c9838396556b809b74c30.

MS151994 committed:
Add container to show user

MS151994 committed:
Refactor show styles one user

MS151994 committed:
Remove console logs

MS151994 committed:
Remove console log

MS151994 committed:
Add filtering modal

MS151994 committed:
Add search and filter button

MS151994 committed:
update description

MS151994 committed:
Add filter box to component

Fremen1990 committed:
70. FE - Redux Toolkit - user update PATCH reducer

Fremen1990 committed:
70. FE - Redux Toolkit - user update PATCH reducer - removed all console.logs

Fremen1990 committed:
Merge pull request #40 from Fremen1990/feature/70-Redux-Toolkit-User-Update

70. FE - Redux Toolkit - user update PATCH reducer

MS151994 committed:
Fix filter styles

MS151994 committed:
Add dispatch method to hr function button

agencik007 committed:
Merge branch 'develop' into feature/68.HR_Student_List_API

agencik007 committed:
Merge pull request #39 from Fremen1990/feature/68.HR_Student_List_API

Feature/68.hr student list api

agencik007 committed:
Getting candidates from database is now working.

agencik007 committed:
Merge branch 'develop' into feature/8.students-page

MS151994 committed:
Add user role to user box

MS151994 committed:
Add default logo

MS151994 committed:
Fix typo

MS151994 committed:
typo from description file

Fremen1990 committed:
RTK cleanup and types from BE configuration

Fremen1990 committed:
Merge pull request #43 from Fremen1990/feature/8.students-page

Feature/8.students page

Fremen1990 committed:
Merge pull request #45 from Fremen1990/feature/redux-toolkit-cleanup

RTK cleanup and types from BE configuration

Fremen1990 committed:
Merge pull request #44 from Fremen1990/feature/Refacor_header_component

Feature/refacor header component

Fremen1990 committed:
Changes for client DEMO

Fremen1990 committed:
Changes for client DEMO

MS151994 committed:
refactor file name and location

MS151994 committed:
Fix import

mateuszslowinski committed:
Merge pull request #46 from Fremen1990/feature/REFACTOR_v2

refactor file name and location

RAlbert5477 committed:
Added paths for svg

RAlbert5477 committed:
Added paths for svg

RAlbert5477 committed:
Added NotFound component

RAlbert5477 committed:
Added NotFound component

MS151994 committed:
add Spinner component (#48)

Fremen1990 committed:
72-FE Student Portfolio CV

Fremen1990 committed:
72-FE Student Portfolio CV + start rating

Fremen1990 committed:
72-FE Student Portfolio CV + start rating + color correction

Fremen1990 committed:
72-FE Student Portfolio CV + start rating + color correction

Fremen1990 committed:
72-FE Student Portfolio CV + start rating + color correction + depressurization  & btn hover

Fremen1990 committed:
Merge pull request #50 from Fremen1990/feature/72-FE-Student-Profile

72-FE Student Portfolio CV

Fremen1990 committed:
75-FE- Student Update profile CV

Fremen1990 committed:
75-FE- Student Update profile CV

Fremen1990 committed:
75-FE- Student Update profile CV + change in enums to small letters per value

Fremen1990 committed:
Merge pull request #51 from Fremen1990/feature/73-FE-Student-Update-Profile-CV

75-FE- Student Update profile CV

RAlbert5477 committed:
Refactored component and style

RAlbert5477 committed:
Refactored component and style

RAlbert5477 committed:
Merge remote-tracking branch 'origin/feature/47.not-found-page' into feature/47.not-found-page

Fremen1990 committed:
Merge pull request #52 from Fremen1990/feature/47.not-found-page

Feature/47.not-found-page

Fremen1990 committed:
First login student - update profile

Fremen1990 committed:
Protected Route correction and Unauthorized styling

agencik007 committed:
Added routes for forgot pwd and change pwd

agencik007 committed:
Added logo for reset pwd

agencik007 committed:
Added pages views for reset pwd

agencik007 committed:
updated constatns - description and routes

agencik007 committed:
Added components for reset pwd and updated Login page

agencik007 committed:
Added actions for send reset token and change password

Commit:
Validation to number input in filter.

agencik007 committed:
Modified some actions, added reducers

Fremen1990 committed:
Admin Reducer and asyncThunks

MS151994 committed:
add simple logic (#49)

* add simple logic

* Prepare function get by filter button from navigate component

* Remove Console log

* add action to get interviews students

* add slice interviews to hr

* add function to check witch button is active and add fetching selector

* refactor navigation buttons

* Add refresh task to props

* Refactor typo in hr actions

* Add interview dispatch

* Refactor typo in result container

* Add refresh students tab after add to interview

* Add action remove from toReview list

* Add action hired student

* fix ts paths

* Add function to buttons

* Refactor function name

* Add correct data to component

Fremen1990 committed:
Admin Functionalities

MS151994 committed:
Feature/add searching students (#55)

* Add searching value and Spinner

* Add searching function

* Add searching results

Commit:
Create CandidateCv page.

Commit:
Added routes to candidate cv page

Commit:
Created DescriptionCandidateCvBox.

Commit:
Created SubtitlesCandidateSection.

Commit:
Created AsideCandidateCvBox.

Commit:
Created structure RatingCandidateCvBox and RatingViewBox and styles.

Commit:
Created structure ExpectationCandidateBox and styles.

Commit:
Created structure LinksCandidateBox and styles.

Commit:
Added styles and structure for CandidateCv.

Commit:
Added second button in AsideCandidateCvBox.

Fremen1990 committed:
Admin Functionalities - One Student and import HR init

Commit:
Take candidate from database and pass data to components.

Commit:
Added display of specific Cv candidate on the hr page.

Commit:
Handled hired and disinterest buttons with appropriate methods.

Commit:
Used types from backend.

Commit:
Changed types of values in filter component.

Commit:
Simple refactor of code.

mateuszslowinski committed:
Merge branch 'develop' into feature/Hr-page-filter-logic

RAlbert5477 committed:
Added profile hr component
Feature/78.hr-page

mateuszslowinski committed:
Merge pull request #53 from Fremen1990/feature/Hr-page-filter-logic

Validation for filter modal.

mateuszslowinski committed:
Merge pull request #56 from Fremen1990/feature/candidate-portfolio-page

Feature/candidate portfolio page

agencik007 committed:
Added margin-bottom to Logo for better view

agencik007 committed:
Changed pages

agencik007 committed:
Changes in components

agencik007 committed:
Added descriptions

agencik007 committed:
Added and modified redux thunks

agencik007 committed:
Merge branch 'develop' into feature/75.change-password

RAlbert5477 committed:
Added style
Feature/78.hr-page

RAlbert5477 committed:
Added style to component
Feature/78.hr-page

Fremen1990 committed:
Admin Functionalities - v0.1 beta

MS151994 committed:
Add rreact pagination

MS151994 committed:
Add pagination to component

MS151994 committed:
Add style pagination buttons

Fremen1990 committed:
Merge branch 'develop' into feature/71-FE-Redux-Toolkit-Admin-Reducer

agencik007 committed:
Merge pull request #58 from Fremen1990/feature/71-FE-Redux-Toolkit-Admin-Reducer

Feature/71 fe redux toolkit admin reducer

RAlbert5477 committed:
Merge pull request #57 from Fremen1990/feature/78.hr-page

Feature/78.hr-page

MS151994 committed:
Add searching modal component and style

MS151994 committed:
Reactor filter section

MS151994 committed:
Refactor HR page

Fremen1990 committed:
Admin Functionalities - v1 beta - Toast Container configuration and numerics in data tables

Fremen1990 committed:
Merge remote-tracking branch 'origin/feature/71-FE-Redux-Toolkit-Admin-Reducer' into feature/71-FE-Redux-Toolkit-Admin-Reducer

Fremen1990 committed:
Merge pull request #54 from Fremen1990/feature/75.change-password

Feature/75.change password

Fremen1990 committed:
Merge pull request #60 from Fremen1990/feature/71-FE-Redux-Toolkit-Admin-Reducer

Feature/71 fe redux toolkit admin reducer

Fremen1990 committed:
Small changes after pull requests mess

Fremen1990 committed:
Merge pull request #59 from Fremen1990/feature/hr_pagination_students

Feature/hr pagination students

Fremen1990 committed:
Small changes after pull requests mess

Fremen1990 committed:
File-saver and example import files to be downloaded

Commit:
Rwd for CandidateCv page.

Commit:
Rwd for student cv modal.

mateuszslowinski committed:
Merge pull request #62 from Fremen1990/feature/Rwd-for-cv

Feature/rwd for cv

Commit:
Remove header for no role user.

Commit:
Added view account for hr in header and added option redirect to change password in button.

Commit:
Refactor validation of data from filter send to backend

Commit:
Created slice and actions for filtered for hr candidate and interviews users.

mateuszslowinski committed:
Merge pull request #63 from Fremen1990/feature/updated-project

Feature/updated project

Commit:
Refactor CandidateCv styles.

Commit:
Replace SubtitlesCandidateSection to SubtitlesSection and refactor.

Commit:
Replace LinksCandidateBox to LinkBox and refactor.

Commit:
Replace DescriptionCandidateCvBox to DescriptionBox and refactor.

Commit:
Refactor styles in ExpectationCandidateBox.

Commit:
Replace RatingCandidateCvBox to RatingBox.tsx and refactor.

Commit:
Refactor AsideCandidateCvBox styles.

Commit:
Changed the order of imports.

Commit:
Changed edit form cv styles and removed email input.

Commit:
Added remove functions for project and portfolio edit form elements.

Fremen1990 committed:
ReadMe Update

Fremen1990 committed:
ReadMe Update

Fremen1990 committed:
ReadMe Update

Fremen1990 committed:
Admin color buttons

Commit:
Back shows header to home page.

Commit:
Handled change student status by student.

Commit:
Changed button text using description constants.

Commit:
Handled changed status button in student portfolio cv.

Commit:
Changed handled student status by new data from backend.

Fremen1990 committed:
Cypress init and first tests

Fremen1990 committed:
Cypress init and first tests

mateuszslowinski committed:
Merge pull request #66 from Fremen1990/Feature/handle-student-status-by-student

Feature/handle student status by student

mateuszslowinski committed:
Merge branch 'develop' into feature/refactor-candidate-cv-page

Fremen1990 committed:
Merge pull request #67 from Fremen1990/TESTING-CYPRESS

Testing cypress

MS151994 committed:
Add value to component

MS151994 committed:
Refactor reducer actions

MS151994 committed:
Refactor results container

MS151994 committed:
Refactor one user box

MS151994 committed:
Refactor filter section

MS151994 committed:
Refactor filtering component

MS151994 committed:
Add functionality to filter form component

MS151994 committed:
Add filtering results modal

RadekJ87 committed:
Merge pull request #68 from Fremen1990/feature/Filter_modal_funcionality

Feature/filter modal funcionality

Fremen1990 committed:
Merge pull request #65 from Fremen1990/feature/refactor-candidate-cv-page

Feature/refactor candidate cv page
